### PR TITLE
Added Bydgoszcz city

### PR DESCRIPTION
**Pull request by BudzioT**

### CODE REVIEW

File: components/Bydgoszcz/About.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <div className="space-y-12 max-w-8xl">
+        <h2 className="pb-6 text-4xl text-center uppercase retro text-whitepurp">
+          Make games, <span className="text-pink">meet cool people</span>
+        </h2>
+        <div className="grid grid-cols-1 xl:grid-cols-2 grid-rows-2 gap-12 leading-[3rem] tracking-wider mx-6 2xl:mx-0">
+          <div className="row-span-2 p-8 space-y-12 border-4 border-dashed border-pink md:p-12">
+            <p>
+              {}
+              Counterspell Bydgoszcz is a beginner friendly game jam for high
+              schoolers happening in Example City where{" "}
```
Comment (samdev-7) [inline components/Bydgoszcz/About.js:13]: You missed this one!

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/bydgoszcz**](https://counterspell-9etmbzzc8.hackclub.dev/bydgoszcz) |
|:-:|
|<a href="https://counterspell-9etmbzzc8.hackclub.dev/bydgoszcz">Screenshot of /bydgoszcz</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/yUNBxfkkQphtOU6JsF5VA8HY9vDppLSq29IHuAwzd-HKW_5RWUBCPkA76yB6l135uSCJATrGFZMV7fKfGi2UHA.png">(view full size)</a> |







Commit da62ffdf834ecb47ccf7a79b33aa825a49eb9595 (https://counterspell-9etmbzzc8.hackclub.dev).

**Review by BudzioT — COMMENTED** [review]
Fixed issue

File: components/Bydgoszcz/About.js (hunk 1 above)
Comment (BudzioT) [reply components/Bydgoszcz/About.js]: Thanks!

**Review by samdev-7 — APPROVED** [review]

**Pull request merged**